corehq/apps/app_manager/templates/app_manager/app_diff.html

```diff
@@ -0,0 +1,49 @@
+{% extends "style/bootstrap3/base_section.html" %}
```

Comment:
It did cross my mind, but It seemed like a small enough amount of reuse that it wouldn't be worth pulling out. I think the shared bits are:
- "see original app" paragraph
- The table, except the for loop is different, and this one uses bootstrap 3 classes instead of 2.
- The js function is the same (minus one line)

So, I didn't see an obvious way to pull out the shared bits without some parameterization. But there definitely might be a good way to do it. Feel free to push back if you think I should think about it harder :)